frontend/src/components/plugins/flows/CustomPivotNode.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Handle, Position, NodeToolbar } from "reactflow";
+import "reactflow/dist/style.css";
+import { Badge } from "reactstrap";
+
+function CustomPivotNode({ data }) {
+  return (
+    <>
+      <NodeToolbar position="top" align="start" isVisible offset={3}>
+        <Badge color="#b5ba66" style={{ backgroundColor: "#b5ba66" }}>
```

Comment:
The rgb code is repeated several times, I think you can define a variable for it (colorConst file)